.github/workflows/integration-tests.yml

```diff
@@ -74,6 +74,7 @@ jobs:
           ("dev") echo "debug";;\
           ("test") echo "debug";;\
           ("bench") echo "release";;\
+          ("integration-test") echo "debug";;\
```

Comment:
You don't need to change the conversion logic. `integration-test` builds into the folder with the same name as the name of profile